moduleroot/CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ $ git checkout -b some-topic-branch
 * Open a pull request for the appropriate project.
 * Contributors will review your pull request, suggest changes, and merge it when it’s ready and/or offer feedback.
 
-If you have questions feel free to contact the [maintainers](https://github.com/intelsdi-x/snap/blob/master/README.md#maintainers).
+If you have questions feel free to contact the [maintainers](https://github.com/intelsdi-x/snap/blob/master/CONTRIBUTING.md).
```

Comment:
@nanliu, what do you think about pointing to `https://github.com/intelsdi-x/snap/blob/master/docs/MAINTAINERS.md`. In CONTRIBUTING.md there is no information how to contact to Snap's maintainers, so I suppose docs/MAINTAINERS.md should be linked instead.